frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -286,9 +290,10 @@ export const ImageManifestVulnList: React.FC<ImageManifestVulnListProps> = (prop
   return (
     <Table
       {...props}
-      aria-label="Image Manifest Vulns"
+      aria-label="Image Manifest Vulnerabilities"
       Header={ImageManifestVulnTableHeader}
       Row={ImageManifestVulnTableRow}
+      EmptyMsg={() => <MsgBox title="No Image Vulnerabilities Found" detail="" />}
```

Comment:
This will unmount and remount the empty msg component on each render. It would be better not to inline.

Reply:
Fixed